lms/djangoapps/grades/tests/utils.py

```diff
@@ -24,17 +24,17 @@ def mock_get_score(earned=0, possible=1):
     Mocks the get_score function to return a valid grade.
     """
     with patch('lms.djangoapps.grades.new.subsection_grade.get_score') as mock_score:
-        mock_score.return_value = ProblemScore(earned, possible, earned, possible, 1, True, None, None)
+        mock_score.return_value = ProblemScore(earned, possible, earned, possible, 1, True, None, None, True)
```

Comment:
This is getting kind of hairy to read - could this use keyword args for values that maybe be unclear instead of ending with the cryptic `1, True, None, None, True)`?